src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -273,10 +276,24 @@ private void PostProcessProjectStatus(IEnumerable<ProjectData> projects)
             };
 
             // Find projects with different paths within the same group
-            var projectPathsInGroup = projectsGroupedByGuid
+
+            var projectPathsInGroup = new List<string>();
+
+            if (!runtimeInformationWrapper.IsOS(OSPlatform.Linux) && !runtimeInformationWrapper.IsOS(OSPlatform.OSX))
```

Comment:
Out of interest, why are you testing `not Linux and not OSX`, rather than `is Windows`?

Reply:
No particular reason, that's definitely a bit less complex to check for Windows directly